### PR TITLE
fix: remove ESS role when not mapped to employee

### PR DESCRIPTION
**Pull request by rtdany10**
Currently, only Employee role is removed, leaving behind ESS and giving the user access to pretty much everything(HR&Payroll) without any user permission(unless explicitly set).

### CODE REVIEW

**Comment by rtdany10** [comment]
@ruchamahabal please check

File: erpnext/setup/doctype/employee/employee.js [hunk 1]
```diff
@@ -83,6 +83,7 @@ frappe.ui.form.on("Employee", {
 			},
 			callback: function (r) {
 				frm.set_value("user_id", r.message);
+				frm.save();
```
Comment (rtdany10) [inline erpnext/setup/doctype/employee/employee.js]: Better way would be to set it directly in db and update the form. Thoughts?

Comment (deepeshgarg007) [reply erpnext/setup/doctype/employee/employee.js]: @rtdany10 Can you implement and push in the same PR?

Comment (rtdany10) [reply erpnext/setup/doctype/employee/employee.js]: Updated

**Comment by rtdany10** [comment]
@deepeshgarg007

**Comment by rtdany10** [comment]
@deepeshgarg007 backport also please, to 15 and 14

**Pull request merged**